### ISSUE TITLE
1.0 alias issues

### ISSUE THREAD

**Issue by Ryzant** [ronh3/poopDeck#27]
the followin aliases dont seem to work for me. Mudlet seems to accept the command but nothing happens. I've removed the package. downloaded a fresh package. then reinstalled an below they still didnt seem to work.
Clear Rigging - crig
Douse - dou(rms)- all 3 doesnt work
Ship Rescue - sres
Set Weapon - seaweapon - all 3 weapons

Also observed that the follow alias aren't on poopfull and poopsail or poopmonster
firc - fire chainshot 
showon - Turn shipwarning on
mainn - maintain none
scomm on - Turn on comm screen
wind - invoke windboost
dous - fill douse the SAILS with a bucket 

I'm weeks late on the testing because I only noticed that you put the 1.0 packge back on today.  I haven't tried the autofire yet. I've only tested the aliases.

I appreciate you creating and improving poopDeck!

**Comment by Ryzant** [comment]
I have since redownloaed the 1.0 package and still getting same issues. 
Additionally, not sure if you'd prefer me to create a separate issues but just a couple of issues related to seamonsters. 
1.) The timer isn't working for me. killed a few sea monsters and the timer has not shown up since downloading.
2.) Out of range. During the 1 time I was captain of a sea monster hunt, the pink Out of Range message stays in the prompt even after I've killed a monster.
